chia-tools/graph.gnuplot

```diff
@@ -59,6 +59,19 @@ plot "chain-resource-usage-cdf.dat" using ($5)*8/1000000:1 with lines title "Ato
 "chain-resource-usage-cdf.dat" using ($6)*8/1000000:1 with lines title "Pair size", \
 "chain-resource-usage-cdf.dat" using ($7)/1000000:1 with lines title "Heap size"
 
+set output "blockchain-object-usage-cdf.png"
+
+set title "block object usage"
+set xlabel "count (million)"
+set xrange [0:*]
+
+plot "chain-resource-usage-cdf.dat" using ($5)/1000000:1 with lines title "Allocated atoms", \
+"chain-resource-usage-cdf.dat" using ($6)/1000000:1 with lines title "Allocated pairs"
```

Comment:
this is somewhat independent, I used this to double check that these limits have plenty of margin from what's currently being used by blocks